router/router.go

```diff
@@ -292,6 +311,11 @@ func New(cfg *config.Configuration, rateConvertor *currency.RateConverter) (r *R
 	return r, nil
 }
 
+// Grabbing the same dialer context as the default transport uses
```

Comment:
Comment is misleading. This returns the DialContext of a Dialer as a convenience function. Comment also does not follow Go standards, should start with the function name "defaultTransportDialContext returns..."